### DISCUSSION TITLE
Welcome to Training Discussions!

### DISCUSSION THREAD

**Discussion by armanceau** [armanceau2/Training#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ApDnoze** [comment]
test

**Reply by armanceau** [reply]
ezaufaf

**Reply by ApDnoze** [reply]
test

**Reply by natzan** [reply]
yo

**Reply by natzan** [reply]
> ezaufaf

yo

**Comment by natzan** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

hello
